sharness/t0032-ctl-health.sh

```diff
@@ -21,7 +21,7 @@ test_expect_success IPFS,CLUSTER "health metrics without metric name fails" '
 '
 
 test_expect_success IPFS,CLUSTER "list latest metrics logged by this peer" '
-    pid=`docker exec ipfs sh -c "ipfs id | jq .ID"`
+    pid=`docker exec ipfs sh -c "ipfs id"` | jq ".ID"
```

Comment:
Small typo here

This should give you the pid value
```suggestion
    pid=`docker exec ipfs sh -c "ipfs id" | jq ".ID"`
```

Reply:
This is exactly what I  fixed. You can't run `jq` in the container.

Reply:
ah no, right